### PR TITLE
death nettle nerf

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
death nettle has been nerfed to be at a more station-centric power level- at max potency, it is a fireaxe sidegrade with 25 damage and 12 throwforce, one handed, that cannot penetrate thick clothing, and no random stun
both nettles have had their acid levels nerfed by half as well, due to how easy to produce they are

## Why It's Good For The Game
death nettles have up to 45 damage, 20% chance to stun, and 100% chance to drop all items on hit at the moment, as well as being thrown to apply massive amounts (up to around 50) of fluorosulfiric acid, all while being able to be produced in massive numbers by botanists. this brings them down to non-antag station levels. 

if this is too much of a nerf, ill take feedback from maintainers

## Changelog
:cl:
balance: nerfs death nettles. they are now no more powerful than a fire axe
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/grown/nettle.dm [hunk 1]
```diff
@@ -88,28 +88,22 @@
...
-		to_chat(M, "<span class='danger'>You are stunned by the powerful acid of [src]!</span>")
-		log_combat(user, M, "attacked", src)
-
-		M.adjust_blurriness(force/7)
-		if(prob(20))
-			M.Unconscious(force / 0.3)
-			M.Paralyze(force / 0.75)
-		M.drop_all_held_items()
+/obj/item/reagent_containers/food/snacks/grown/nettle/death/attack(mob/living/M, mob/user)
+	if(M.can_inject(user)
```
Comment (BrumalLake) [inline code/modules/hydroponics/grown/nettle.dm]: unpaired 😢

File: code/modules/hydroponics/grown/nettle.dm [hunk 2]
```diff
@@ -88,28 +88,22 @@
...
-		M.adjust_blurriness(force/7)
-		if(prob(20))
-			M.Unconscious(force / 0.3)
-			M.Paralyze(force / 0.75)
-		M.drop_all_held_items()
+/obj/item/reagent_containers/food/snacks/grown/nettle/death/attack(mob/living/M, mob/user)
+	if(M.can_inject(user))
+		return FALSE
+	else
+		return ..()
```
Comment (Archanial) [inline code/modules/hydroponics/grown/nettle.dm]: doesn't parrent actually run hitting code and check for things like range?
```suggestion
/obj/item/reagent_containers/food/snacks/grown/nettle/death/attack(mob/living/M, mob/user)
	if(!..())
		return
	if(M.can_inject(user))
		return FALSE
```

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/nettle.dm (hunk 2 above)
Comment (zeskorion) [reply code/modules/hydroponics/grown/nettle.dm]: Will check when home

Comment (zeskorion) [reply code/modules/hydroponics/grown/nettle.dm]: no, this is not how the attack proc works. i did make a mistake though, and  i rectified it

**Comment by Wilchenx** [comment]
>both nettles have had their acid levels nerfed by half as well

doesn't that basically do nothing as all plants are also reagent containers, so you can essentially fill them up with syringes?

**Review by Archanial — APPROVED** [review]

**Comment by zeskorion** [comment]
> > both nettles have had their acid levels nerfed by half as well
> 
> doesn't that basically do nothing as all plants are also reagent containers, so you can essentially fill them up with syringes?

it makes them harder to function as mass produced armor deleting throwing weapons

**Comment by PerishedFraud** [comment]
Remove the nettle completely. There's no reason for a botanist to have it to begin with. Remove lemon bombs too.  If people want to weaponize botany at least force them to be creative.

**Review by Archanial — APPROVED** [review]

**Review by francinum — APPROVED** [review]

**Pull request merged**